examples/application/evaluate_gaming_traffic.cc

```diff
@@ -73,6 +74,7 @@ CalculateThroughput (Ptr<OutputStreamWrapper> stream)
 {
   Time now = Simulator::Now ();    /* Return the simulator's virtual time. */
   double cur = (packetSink->GetTotalRx () - lastTotalRx) * 8.0 / computeThroughputPeriodicity.GetSeconds () / 1e3;     /* Convert Application RX Packets to kb. */
+  averageThroughput = packetSink->GetTotalRx () * 8.0 / now.GetSeconds () / 1e6; /* Average throughput in Mb */
```

Comment:
it is not necessary to compute it multiple times. just compute it once at the end